flake.nix

```diff
@@ -33,7 +33,15 @@
         };
         env-with-sim = pkgs.mkShell {
           name = "sonata-sw-with-sim";
-          packages = default.nativeBuildInputs ++ [sonataSystemPkgs.sonata-simulator];
+          packages =
+            (with sonataSystemPkgs; [
+              sonata-simulator
+              sonata-sim-boot-stub
```

Comment:
```suggestion
```

Reply:
Don't we need this though? The `sonata-sim-boot-stub` contains the sim_boot_stub binary we need to run simulations.

Reply:
The `packages` is for exposing to `PATH`. The `shellHook` can reference to any packages without it being present in `packages`.